string-manipulation-2/src/capitalize-word.js

```diff
@@ -0,0 +1,16 @@
+/* exported capitalizeWord */
+function capitalizeWord(word) {
+  var lowerCase = word.toLowerCase();
+  var seperateWord = lowerCase.slice(1, 4);
+  var firstLetter = word[0];
+  var upperCase = firstLetter.toUpperCase();
+  if (word.length > 4) {
```

Comment:
Just check if it is 'javascript' 😄

Reply:
![Screen Shot 2021-03-15 at 9 25 51 PM](https://user-images.githubusercontent.com/74885918/111255845-11514880-85d5-11eb-8f9b-d55ead1558f7.png)